### PR TITLE
[github-ci] Add Auto-YAPF CI to the Sibyl System codebase

### PR DESCRIPTION
**Pull request by AndreiJirohHaliliDev2006**
This merge request contains the addition of the `.github/workflows/yapf.yml` file adopted from <https://github.com/AnimeKaizoku/SaitamaRobot/blob/shiken/.github/workflows/yapf.yml>, with some customizations made by myself.

### CODE REVIEW

**Comment by TsunayoshiSawada** [comment]
@AndreiJirohHaliliDev2006 @Dragsama

File: .github/workflows/yapf.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Format Python code
+on: 
+  push: 
+    branches: [master]
+  pull_request:
+    branches: [master]
```
Comment (AndreiJirohHaliliDev2006) [inline .github/workflows/yapf.yml:6]: Should I send PRs on Auto-YAPF against `master`, @Dragsama?

**Comment by AndreiJirohHaliliDev2006** [comment]
> @AndreiJirohHaliliDev2006 @DragSama

Uhmm, why Tsuna?

**Comment by DragSama** [comment]
Sibyl doesn't need it imo

**Comment by DragSama** [comment]
Since I autopep8 code every few week

**Pull request closed without merging**